zstd_stream_test.go

```diff
@@ -19,10 +20,14 @@ func failOnError(t *testing.T, msg string, err error) {
 	}
 }
 
-func testCompressionDecompression(t *testing.T, dict []byte, payload []byte) {
+func testCompressionDecompression(t *testing.T, dict []byte, payload []byte, nbWorkers int) {
 	var w bytes.Buffer
 	writer := NewWriterLevelDict(&w, DefaultCompression, dict)
-	_, err := writer.Write(payload)
+
+	err := writer.SetNbWorkers(nbWorkers)
```

Comment:
I would probably only call this is `nbWorkers` > 1 so you can test the (most common) path where the user doesn't call this method

Reply:
I think this is addressed.